tests/RedisTest.php

```diff
@@ -5461,8 +5461,17 @@ private function generateSessionId()
     {
         if (function_exists('session_create_id')) {
             return session_create_id();
+        } else if (function_exists('random_bytes')) {
+            return bin2hex(random_bytes(8));
+        } else if (function_exists('openssl_random_pseudo_bytes')) {
+            return bin2hex(openssl_random_pseudo_bytes(8));
         } else {
-            $encoded = bin2hex(openssl_random_pseudo_bytes(8));
+            /* Not cryptographically secure, but it doesn't need to be
+             * for the test.  We just want an ID */
+            $encoded = '';
```

Comment:
I think using `uniqid` will be simpler here :)

Reply:
Yeah, that would be better.  It wasn't clear to me if the test needed 8 bytes specifically for some reason 😄